contracts/control/unstructured/CrossChainOwnable/UCrossChainOwnable.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.13;
+
+import "@openzeppelin/contracts/crosschain/CrossChainEnabled.sol";
+import "../UOwnable.sol";
+import "../../../storage/UStorage.sol";
+
+/**
+ * @title UCrossChainOwnable
+ * @notice Library to manage the cross-chain ownership lifecycle of upgradeable contracts.
+ * @dev This contract has been extended from the Open Zeppelin library to include an
+ *      unstructured storage pattern so that it can be safely mixed in with upgradeable
+ *      contracts without affecting their storage patterns through inheritance.
```

Comment:
Can we add another sentence here about the recommended usage pattern? basically explaining how `_crossChainRestricted` works?